lib/services/android-project-service.ts

```diff
@@ -52,6 +52,7 @@ class AndroidProjectService extends projectServiceBaseLib.PlatformProjectService
 					`${this.$projectData.projectName}-release.apk`
 				],
 				frameworkFilesExtensions: [".jar", ".dat", ".so"],
+				frameworkVersion: require(this.$projectData.projectFilePath).nativescript["tns-ios"].version,
```

Comment:
also consider using `this.$fs.readJson` instead of require, as the require is caching the results
one more thing - won't this fail in case `tns-ios` does not exist in nativescript key?